components/script_traits/script_msg.rs

```diff
@@ -209,3 +210,10 @@ pub enum SWManagerMsg {
     /// Provide the constellation with a means of communicating with the Service Worker Manager
     OwnSender(IpcSender<ServiceWorkerMsg>),
 }
+
+/// Messages outgoing from the File Manager thread to constellation
+#[derive(Deserialize, Serialize)]
+pub enum FileManagerMsg {
+    /// Requesting to open file select dialog
+    OpenFileSelectDialog(Vec<FilterPattern>, bool, IpcSender<Option<Vec<String>>>)
+}
```

Comment:
I might have missed something, but is that message actually used somewhere?

Reply:
No, I guess it's my mistake leftover while refactoring. Thanks for catching.